src/button.tsx

```diff
@@ -88,6 +98,9 @@ const renderButtonContent = ({
     setShouldRenderSpinner,
     renderText,
     textContentStyle,
+    iconLeft,
+    iconRight,
+    withChevron,
```

Comment:
`withChevron` is not showing on playroom props panel 🤔

Reply:
fixed